main.rb

```diff
@@ -0,0 +1,19 @@
+require './my_enumerable'
```

Comment:
Please use `require_relative` to import `MyEnumerable` since `require` should be used for external files, like gems, while `require_relative` should be used for referring to files within your directory.